src/integ_test/java/games/strategy/engine/lobby/server/db/EmailLimitIntegrationTest.java

```diff
@@ -29,28 +29,26 @@ public static void setup() throws SQLException {
   }
 
   @Test
-  public void testAllowsMaximumLentgh() {
+  public void testAllowsMaximumLentgh() throws Exception {
     createAccountWithEmail(getStringWithLength(60) + "@" + getStringWithLength(193));
   }
 
   @Test
-  public void testAllowsMaximumLocalLength() {
+  public void testAllowsMaximumLocalLength() throws Exception {
     createAccountWithEmail(getStringWithLength(64) + "@" + getStringWithLength(189));
   }
 
   private static String getStringWithLength(final int length) {
     return Strings.padStart(Util.createUniqueTimeStamp(), length, 'a');
   }
 
-  private static void createAccountWithEmail(final String email) {
+  private static void createAccountWithEmail(final String email) throws SQLException {
     try (PreparedStatement ps =
         connection.prepareStatement("insert into ta_users (username, email, password) values (?, ?, ?)")) {
       ps.setString(1, Util.createUniqueTimeStamp());
       ps.setString(2, email);
       ps.setString(3, MD5Crypt.crypt("password"));
       ps.execute();
-    } catch (final SQLException e) {
```

Comment:
Why isn't this method wrapped in the assertNotThrows method?

Reply:
Yes, you're right.  There should be an explicit assertion someplace because failure is indicated if the database raises an exception.  For some reason (a derp most likely), I was under the impression the database code was secondary to the test.  I will submit a PR in a few minutes to correct this.